google/cloud/aiplatform/training_jobs.py

```diff
@@ -1620,9 +1760,29 @@ def run(
         Any of ``training_fraction_split``, ``validation_fraction_split`` and
         ``test_fraction_split`` may optionally be provided, they must sum to up to 1. If
         the provided ones sum to less than 1, the remainder is assigned to sets as
-        decided by Vertex AI.If none of the fractions are set, by default roughly 80%
+        decided by Vertex AI. If none of the fractions are set, by default roughly 80%
         of data will be used for training, 10% for validation, and 10% for test.
 
+        Data filter splits:
+        Assigns input data to training, validation, and test sets
+        based on the given filters, data pieces not matched by any
+        filter are ignored. Currently only supported for Datasets
+        containing DataItems.
+        If any of the filters in this message are to match nothing, then
+        they can be set as '-' (the minus sign).
+        Supported only for unstructured Datasets.
+
+        Predefined splits:
+        Assigns input data to training, validation, and test sets based on the value of a provided key.
```

Comment:
```suggestion
        Predefined split requires ``predefined_split_column_name`` is provided.
        Assigns input data to training, validation, and test sets based on the value of a provided key.
```

Please apply this to class docstrings that support predefined splits.